static-build/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ cmake_minimum_required(VERSION 3.1)
 # linux machine).
 project(tarantool-static C CXX)
 
+include(FindPackageMessage)
```

Comment:
I don't see how it is used.

Reply:
Got it, it is in `os.cmake`. Maybe it is better to move the include to `os.make`, but I don't insist.